clusters/build-clusters/02_cluster/installation_params.md

```diff
@@ -0,0 +1,60 @@
+# Build02: installation parameters
+
+## [Configuring a GCP project](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html)
+
+I will ask team-dpp for help on this, as we did this for ci-infra on aws for `build01`.
+
+Use project "openshift-ci-infra" as we do for `api.ci`.
+
+### [Configuring DNS for GCP](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html#installation-gcp-dns_installing-gcp-account)
+
+* base-domain: e.g, the hostname for image registry is `<route_name>.apps.<cluster_name>.<base_domain>`, where
+
+    * `<route_name>` is configurable after installation,
+    * `<cluster_name>` is `build02`,
+    * `<base_domain>` is a public hosted zone for your domain or subdomain in your GCP project. 
```

Comment:
@bbguimaraes 
IIRC, you have some request on the hostname of the registry.
I think it is the time that you express your opinion again.
What is the hostname do you want to see for `build02`?
It might be easier to do it now than post-installation.

Reply:
Finally I got some doc on specify `route.spec.host` as a customized domain.
https://cloud.ibm.com/docs/openshift?topic=openshift-openshift_routes

I did not find anything from openshift official doc.

Reply:
So if we only care image.registry's router, we could probably do it as day2 operation.
I guess we could do it similarly to the domain of api server (which is an alias on aws-route53 of an aws loadbalancer).

<img width="911" alt="Screen Shot 2020-05-06 at 9 51 17 AM" src="https://user-images.githubusercontent.com/4013349/81185097-2a35aa00-8f7f-11ea-91a8-9dbd867ab350.png">